engine/source/engine.cpp

```diff
@@ -114,7 +109,7 @@ ModuleTickOrder OldEngine::Init(Engine& engine)
     applicationModule.GetInputManager().GetMousePosition(mousePos.x, mousePos.y);
     _lastMousePos = mousePos;
 
-    _ecs->GetSystem<PhysicsSystem>()->InitializePhysicsColliders();
+    //_ecs->GetSystem<PhysicsSystem>()->InitializePhysicsColliders();
```

Comment:
it just doesnt work with the asset, it also isnt something that we should use since, it just spawns cube colliders around every mesh and that doesnt make sense in our scene.